### PR TITLE
Improve members UI

### PR DESCRIPTION
**Pull request by bryce-fitzsimons**
### Features and Changes

- improve members ui
- add section descriptions
- show project roles for invites
- clarify seat calcs for invites

Screenshot 2024-03-19 at 2 07 20 PM

### CODE REVIEW

File: packages/front-end/components/Settings/Team/PendingMemberList.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { FC, useState } from "react";
+import React, { FC, useState } from "react";
```
Comment (bryce-fitzsimons) [inline packages/front-end/components/Settings/Team/PendingMemberList.tsx:1]:
```suggestion
import { FC, useState } from "react";
```

File: packages/front-end/pages/settings/team/[tid].tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import router from "next/router";
-import { FC, useState } from "react";
+import React, { FC, useState } from "react";
```
Comment (bryce-fitzsimons) [inline packages/front-end/pages/settings/team/[tid].tsx:2]:
```suggestion
import { FC, useState } from "react";
```

**Comment by august-growthbook** [comment]
Thanks for the improvements!

File: packages/front-end/components/Settings/Team/InviteList.tsx [hunk 3]
```diff
@@ -105,8 +109,13 @@ const InviteList: FC<{
     <div>
       <h5>Pending Invites{` (${invites.length})`}</h5>
       <div className="text-muted mb-2">
-        Invites that have been sent but have not yet been accepted.{" "}
-        <strong>Invited users count towards plan seat limits.</strong>
+        Invites that have been sent but have not yet been accepted.
+        {actualPlan !== "enterprise" && (
```
Comment (tzjames) [inline packages/front-end/components/Settings/Team/InviteList.tsx:113]: I think you should keep this blurb for everyone.  All enterprise plans that I know of have a seat limit.  Up until recently though we haven't been enforcing it, and we have been relying on our enterprise customers to tell us when they go over, and we haven't even been letting them know that they are over in any obvious way.  But now we have a hardCap on some enterprise licenses we have been giving out, and there is a PR in review where we will let everyone know in the AccountPlanNotices in the top nav bar when they go over the limit whether they have a hard cap or not.  So I think this is useful information regardless.

File: packages/front-end/components/Settings/Team/InviteList.tsx [hunk 4]
```diff
@@ -161,6 +178,26 @@ const InviteList: FC<{
                 <td>{email}</td>
                 <td>{datetime(dateCreated)}</td>
                 <td>{roleInfo.role}</td>
+                {!project && (
+                  <td className="col-3">
+                    {member.projectRoles?.map((pr) => {
+                      const p = projects.find((p) => p.id === pr.project);
+                      if (p?.name) {
```
[the remark this answers is not in the document]
Comment (bryce-fitzsimons) [reply packages/front-end/components/Settings/Team/InviteList.tsx:176]: looks like its just a sanity check on valid project

**Pull request merged**